repair/repair.cc

```diff
@@ -1207,55 +1206,8 @@ future<int> repair_service::do_repair_start(gms::gossip_address_map& addr_map, s
             if (nr_vnode_table != 0) {
                 throw std::invalid_argument("Mixed vnode table and tablet table");
             }
-            is_tablet = true;
             throw tablets_unsupported();
```

Comment:
I think we can even remove `nr_tablet_table` and `nr_vnode_table` and in the loop above, just throw where we increment `nr_tablet_table` now.